assets/helpers/paymentIntegrations/newPaymentFlow/readerRevenueApis.js

```diff
@@ -177,6 +177,9 @@ function postRegularPaymentRequest(
       if (response.status === 500) {
```

Comment:
So just to confirm I've understood correctly, when this PR is merged, we'll return:

```
case 'internal_error':
  return 'Sorry, something has gone wrong. Please try again, or come back later.';
```

And then we'll start displaying this friendlier error after a subsequent PR is merged:

```
case 'personal_details_incorrect':
  return 'Please double check the name and contact details you provided and try again.';
```

Right now the user experience is very poor (I expect they'll see the pending thankyou page, and then get a failure email later), so even this interim change is still a big improvement 👍

Reply:
That's right!